### PR TITLE
Add poll to adjust the Dust Parameter

### PR DESCRIPTION
**Pull request by jiecut42**
@LongForWisdom

### CODE REVIEW

File: governance/polls/Proposal - Adjust the Dust Parameter [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Adjust the Dust Parameter - August 10, 2020
+summary: Signal your support for increasing the Dust Parameter from 20 DAI to 100 DAI
```
Comment (LongForWisdom) [inline governance/polls/Proposal - Adjust the Dust Parameter:3]: Can we have 'for all vault types.' In here as well, please?

File: governance/polls/Proposal - Adjust the Dust Parameter [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (LongForWisdom) [inline governance/polls/Proposal - Adjust the Dust Parameter:1]: Please ensure the filename has the .md suffix.

File: governance/polls/Proposal - Adjust the Dust Parameter (hunk 1 above)
Comment (jiecut42) [reply governance/polls/Proposal - Adjust the Dust Parameter]: 👍

File: governance/polls/Proposal - Adjust the Dust Parameter (hunk 2 above)
Comment (jiecut42) [reply governance/polls/Proposal - Adjust the Dust Parameter]: 👍

**Pull request merged**